db/comdb2.c

```diff
@@ -768,6 +768,8 @@ int64_t gbl_temptable_spills;
 
 int gbl_osql_odh_blob = 1;
 
+int gbl_clean_exit_on_sigterm = 0;
```

Comment:
Should be 1 by default.

Reply:
It ends up being on anyway because we set up signal handling before we set up tunables.  But certainly less confusing to set it to the default value.